planner_reasoner/rdf/base/pp_oslc_shapes.ttl

```diff
@@ -0,0 +1,219 @@
+##############################################################################
+# studied from the bnf specification of PDDL 2.1.
+# https://www.jair.org/media/1129/live-1129-2132-jair.pdf without durative actions
+##############################################################################
+
+@prefix : <http://ontology.cf.ericsson.net/planning_ontology_oslc_shapes#> .
+@prefix ppos: <http://ontology.cf.ericsson.net/planning_ontology_oslc_shapes#> .
+@prefix pp: <http://ontology.cf.ericsson.net/planning_ontology#> .
+@prefix wd: <http://ontology.cf.ericsson.net/warehouse_domain_and_problem#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix oslc: <http://open-services.net/ns/core#> . #oslc core
+@prefix sh: <http://www.w3.org/ns/shacl#> . # shapes constraint languge SHACL
+@base <http://ontology.cf.ericsson.net/planning_ontology_oslc_shapes> .
+
+
+:ActionShape rdf:type oslc:ResourceShape ;
+            oslc:describes :Action ;
+            oslc:property [a oslc:Property ;
```

Comment:
`oslc:name` is a mandatory field for every property that should contain second part of property's Prefixed Name.
For example here it should be: `oslc:name "hasPrecondition"^^xsd:string`.